api/dependencies/file.py

```
@@ -90,158 +126,148 @@ def detect_mime_type_fallback(file_content: bytes, filename: str='') ->str:
 
 
-def check_for_malicious_content(file_content: bytes, filename: str) ->bool:
+def check_for_malicious_content(file_content: bytes, filename: str) -> bool:
     """Check for potentially malicious content patterns."""
     _, ext = os.path.splitext(filename.lower())
     if ext in DANGEROUS_EXTENSIONS:
         return False
-    if file_content.startswith(b'PK'):
+    if file_content.startswith(b"PK"):
         for pattern in MALWARE_BYTE_PATTERNS[:3]:
             if pattern.encode() in file_content:
```

Pylint (3.3.8)

[error] Instance of 'bytes' has no 'encode' member

(E1101)